SteamKit2/SteamKit2/Networking/Steam3/WebSocketContext.cs

```diff
@@ -88,9 +88,8 @@ async Task RunCore(TimeSpan connectionTimeout, CancellationToken cancellationTok
                 }
             }
 
-            public async Task SendAsync(byte[] data)
+            public async Task SendAsync(ArraySegment<byte> segment)
             {
-                var segment = new ArraySegment<byte>(data, 0, data.Length);
                 try
                 {
                     await socket.SendAsync(segment, WebSocketMessageType.Binary, true, cts.Token).ConfigureAwait(false);
```

Comment:
This api does not have a Span<> overload forcing the use of ArraySegment everywhere, which I don't like. Any ideas?